.github/README.md

```
@@ -1,18 +1,77 @@
+#### 2. Install Dependencies
+```bash
+# Install Node.js dependencies
+npm install
+
+# Install PHP/Composer dependencies
+composer install
+```
+
+#### 3. Generate your theme
+
+**Interactive prompts (easiest)**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)